src/Data/LLVM/BitCode/Assert.hs

```diff
@@ -22,7 +22,6 @@ module Data.LLVM.BitCode.Assert
   -- ** Types
   , elimPtrTo
   , elimPtrTo_
-  , ptrTo
```

Comment:
Can we provide some porting instructions (in this PR is fine) for code which might be using `ptrTo`?  What should be done in that code?

Reply:
Yes, I can include some general advice about how to port code to adapt to opaque pointers. Come to think of it, we will need to make a new Hackage release of `llvm-pretty-bc-parser` at some point (see https://github.com/GaloisInc/crucible/issues/1077), so we should create a changelog and mention these changes there.